mapper.go

```diff
@@ -58,7 +59,7 @@ func (m *metricMapper) initFromString(fileContents string) error {
 
 		switch state {
 		case SEARCHING:
-			if line == "" {
+			if line == "" || line[0] == COMMENT_CHAR {
```

Comment:
I'll try a few things. Write some tests. Maybe it's that simple. Your desire to make comments work everywhere rather than just as headers to a definition is fair.